src/jquery.floatThead.js

```diff
@@ -95,6 +95,10 @@
         return result;
       };
     };
+    that.isFunction = function(obj) {
```

Comment:
I don't understand how. There are several calls to util.isFunction for which I was getting isFunction not defined using the latest release and  with a build I did from the latest code. I am not using underscore or lodash. When I added isFunction, everything started to work.